webapi/src/main/scala/org/knora/webapi/store/triplestore/upgrade/plugins/UpgradePluginPR3110.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2021 - 2024 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.store.triplestore.upgrade.plugins
+
+import org.eclipse.rdf4j.model.vocabulary.RDF
+import org.eclipse.rdf4j.sparqlbuilder.core.SparqlBuilder.`var` as variable
+import org.eclipse.rdf4j.sparqlbuilder.core.query.ModifyQuery
+import org.eclipse.rdf4j.sparqlbuilder.core.query.*
+import org.eclipse.rdf4j.sparqlbuilder.rdf.Rdf
+
+import org.knora.webapi.messages.OntologyConstants.KnoraAdmin.KnoraAdminPrefixExpansion
+import org.knora.webapi.slice.admin.AdminConstants
+import org.knora.webapi.slice.admin.repo.rdf.Vocabulary
+import org.knora.webapi.store.triplestore.upgrade.GraphsForMigration
+import org.knora.webapi.store.triplestore.upgrade.MigrateSpecificGraphs
+
+/**
+ * After removing `knora-admin:Institution` class and its properties from the knora-admin ontology this cleans up the DB.
+ * Removes all `knora-admin:Institution` classes from the `knora-admin` named graph.
+ */
+class UpgradePluginPR3110() extends AbstractSparqlUpdatePlugin {
+
+  override def graphsForMigration: GraphsForMigration =
+    MigrateSpecificGraphs.from(AdminConstants.adminDataNamedGraph)
+
+  override def getQuery: ModifyQuery = {
+    var (s, p, o) = (variable("s"), variable("p"), variable("o"))
+    Queries
+      .MODIFY()
+      .prefix(Vocabulary.KnoraAdmin.NS, RDF.NS)
+      .delete(s.has(p, o))
+      .where(
+        s.isA(Rdf.iri(KnoraAdminPrefixExpansion, "Institution"))
+          .andHas(p, o)
+          .from(Vocabulary.NamedGraphs.knoraAdminIri),
+      )
+  }
```

Comment:
**Question (non-blocking):** Just for my understanding: This would remove any institution from an object which is linked to an institution (like `?someProject :belongsToInstitution ?institution`). But it would not remove the institution itself from the DB, right? Do we have any instances of institutions in our DB? Do we want to keep them?

Reply:
You are raising a good point. 

This query removes a triples belonging to an `Institution` class.
I have missed to remove the `?project knora-admin:belongsTo ?s` triples as well. Will add it to the PR.

Reply:
well... I wrongly read it the other way around... but you're right, and I'm glad that my confusion helped to identify a real issue :)

Reply:
see [3c86d84](https://github.com/dasch-swiss/dsp-api/pull/3109/commits/3c86d8475f05e2250fbc45cad9040a3a6d0aa5c2)